### ISSUE TITLE
Long strip view

### ISSUE THREAD

**Issue by RandomPotato26** [d2phap/ImageGlass#525]
Possibly add functionality to view files in a long strip, instead of individually.
I work a fair amount on manga/manhwa translation and would like to read the chapter as it might be shown online while I check for mistakes.

**Comment by d2phap** [comment]
Hi @RandomPotato26 

I guess you want to change mouse wheel actions? ImageGlass supports vertical / horizontal scrolling

![image](https://user-images.githubusercontent.com/3154213/57580154-7ec35800-74d8-11e9-88eb-0e349f446c8c.png)

**Comment by fire-eggs** [comment]
No, he wants a continuous scroll of images. I.e. instead of each image being treated as a separate page (using next/prev to move through 'pages'), all the images are to be shown one on top of another as a scrollable "single page".

A completely arbitrary [example](https://www.webtoons.com/en/fantasy/eleceed/episode-1/viewer?title_no=1571&episode_no=1) .

**Comment by RandomPotato26** [comment]
exactly what @fire-eggs said, honestly just adding more viewing options (side by side, overlay, ect.) would be kinda cool to see as well.

**Comment by LittleDaoist** [comment]
Any update for this?

**Comment by d2phap** [comment]
@LittleDaoist 
Not in my plan yet, probably all new features.
I am focusing on the new WPF version currently.

**Comment by Hccake** [comment]
Looking forward to this feature so I can read manga with ImageGlass. 😻

**Comment by testpillow** [comment]
Also hoping for a continuous scrolling feature to be implemented one day!
